src/Types.hs

```diff
@@ -210,3 +220,37 @@ instance ToJSON PoolOfflineMetadata where
 --instance ToParamSchema PoolOfflineMetadata
 instance ToSchema PoolOfflineMetadata
 
+
+newtype PoolMetadataWrapped = PoolMetadataWrapped Text
+    deriving (Eq, Ord, Show, Generic)
+
+-- Here we are usingg the unsafe encoding since we already have the JSON format
+-- from the database.
+instance ToJSON PoolMetadataWrapped where
+    toJSON (PoolMetadataWrapped metadata) = toJSON metadata
+    toEncoding (PoolMetadataWrapped metadata) = unsafeToEncoding $ encodeUtf8Builder metadata
```

Comment:
This is still returning a plain string that contains encoded JSON. I could expect to be able to do:

```
curl -v http://localhost:3100/api/v1/metadata/{hash} > metadata.json
```

And end up with a valid JSON file. Now however, I need to decode it first! It is encoded to JSON twice!

```hs
λ> Right (String str) <- eitherDecodeFileStrict @Value "metadata.json" 
λ> Right metadata = eitherDecodeStrict @Value (T.encodeUtf8 str)
λ> metadata
Object (fromList [("homepage",String "https://github.com/input-output-hk/cardano-db-sync/tree/master/cardano-db/src/Cardano/Db"),("name",String "test"),("ticker",String "testy"),("description",String "This is a test pool")])
```

Reply:
Why can't you do `curl -v http://localhost:3100/api/v1/metadata/{hash} > metadata.json`?
When I do that, I end up with a valid JSON file.
```
$> curl -v http://localhost:3100/api/v1/metadata/cbdfc4f21feb0a414b2b9471fa56b0ebd312825e63db776d68cc3fa0ca1f5a2f > metadata.json

$> cat metadata.json

{"name": "test", "description": "This is a test pool", "ticker": "testy", "homepage": "https://github.com/input-output-hk/cardano-db-sync/tree/master/cardano-db/src/Cardano/Db"}
```

Reply:
Hmm. Okay, stack trickery at play here, I was still running an old version. Re-compiling didn't help, I had to clear the `.stack-work` and re-compile from scratch. It indeed looks better! 

```console
$ curl -v http://localhost:3100/api/v1/metadata/cbdfc4f21feb0a414b2b9471fa56b0ebd312825e63db776d68cc3fa0ca1f5a2f  | jq
{
  "name": "test",
  "description": "This is a test pool",
  "ticker": "testy",
  "homepage": "https://github.com/input-output-hk/cardano-db-sync/tree/master/cardano-db/src/Cardano/Db"
}
```